### PR TITLE
Homework/Reactweek1

### PR DESCRIPTION
**Pull request by Gohar33**

### CODE REVIEW

**Comment by senner008** [comment]
Good

You have a few issues. Check the dev tools.
- You are referring to an non-existing id on your objects
- Each outer element of your todos should have a key
- You used className wihout assigning it

File: react/week1/TodoList/list/src/Main.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+function Main() {
+    const array = [
+        { name: "Get out of bed", deadline: "Wed Sep 13 2017" },
+        { name: "Brush teeth", deadline: "Thu Sep 14 2017" },
+        { name: "Eat breakfast", deadline: "Fri Sep 15 2017" },
+        { name: "Study code", deadline: "Sat Sep 16 2017" },
+        { name: "Go to shop", deadline: "Sun Sep 17 2017" }
+    ]
+    return (
+        <div className>
```
Comment (senner008) [inline react/week1/TodoList/list/src/Main.js:36]: classname missign assignment

File: react/week1/TodoList/list/src/Main.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            {array.map(item => {
+                return (
+                    <span className="todo-item">
```
Comment (senner008) [inline react/week1/TodoList/list/src/Main.js:39]: missing key on span element

File: react/week1/TodoList/list/src/Main.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        <div>
+            <p>  {props.name}</p>
+        </div >
+    )
+}
+
+
+function Main() {
+    const array = [
+        { name: "Get out of bed", deadline: "Wed Sep 13 2017" },
```
Comment (senner008) [inline react/week1/TodoList/list/src/Main.js:29]: no id here

Comment (Gohar33) [reply react/week1/TodoList/list/src/Main.js:29]: I should add an Id here? 

Thanks a lot for your feedback Niels, I will implement it right away.

**Comment by senner008** [comment]
Thumbs up regarding Separation of concerns/Single responsibility

File: react/week1/TodoList/list/src/Main.js (hunk 3 above)
Comment (senner008) [reply react/week1/TodoList/list/src/Main.js:29]: yes each todo should have a unique id

**Review by senner008 — APPROVED** [review]

**Pull request merged**